src/utils.ts

```diff
@@ -1,21 +1,23 @@
-import { workspace, Disposable, DocumentSelector } from 'vscode';
+import { workspace, Disposable, DocumentSelector, Uri } from 'vscode';
 import { PrettierVSCodeConfig } from './types.d';
 
-let currentRootPath: string = workspace.rootPath;
+let currentRootPath: string = workspace.rootPath!;
 
 export function onWorkspaceRootChange(
     cb: (rootPath: string) => void
 ): Disposable {
+    // TODO: use `workspace.onDidChangeWorkspaceFolders`
```

Comment:
Is this todo still necessary or a future improvement?

Reply:
I think it is for VSCode's multiple workspaces support.

Reply:
Yeah I don't want to tackle too much in this PR.